README.md

```diff
@@ -100,15 +100,137 @@ alt="platformatic"
 
 ## Configuration options
 
+All examples below use **ESM (ECMAScript Modules)** syntax. If you're using **CommonJS (CJS)**, replace the `import`/`export` statements with the following:
+
+  ```js
+  // Replace
+  import neostandard from 'neostandard'
+  export default neostandard({ /* options */ })
+
+  // With
+  const neostandard = require('neostandard')
+  module.exports = neostandard({ /* options */ })
+  ```
+
+Here's a basic example of how to configure NeoStandard:
+
+  ```js
+  import neostandard from 'neostandard'
+
+  export default neostandard({
+    ts: true  // an option
+    // Add other options here
+  })
+  ```
+
+The options below allow you to customize NeoStandard for your project. Use them to add global variables, ignore files, enable TypeScript support, and more.
+
 * `env` - *`string[]`* - adds additional globals by importing them from the [globals](https://www.npmjs.com/package/globals) npm module
+  
+  ```js
+  import neostandard from 'neostandard'
+
+  export default neostandard({
+    env: ['browser', 'node']  // Add browser and Node.js global variables
```

Comment:
`node` is actually added by default (carry over from `standard`, should be changed eventually, see #131 / #130)

```suggestion
    env: ['browser', 'mocha']  // Add browser and mocha global variables
```